examples/FacebookTabsExample/FacebookTabBar.js

```diff
@@ -20,15 +20,31 @@ const FacebookTabBar = React.createClass({
     this._listener = this.props.scrollValue.addListener(this.setAnimationValue);
   },
 
-  setAnimationValue({ value, }) {
-    this.tabIcons.forEach((icon, i) => {
-      const progress = (value - i >= 0 && value - i <= 1) ? value - i : 1;
```

Comment:
It wasn't working the same way when I was using it. I noticed that the GIF on the main page doesn't have this problem (?) but with this formula you'll notice that this only works when value - icon index is between 0 and 1. But when moving to the right it's only the absolute value that falls between 0 and 1, the real number is between 0 and -1. 

When I ran the example here, the icon highlighting to the left was gradual, but the icon moving to the right snapped from grey to blue immediately.